### ISSUE TITLE
More styling and build options in Guttenberg Client View

### ISSUE THREAD

**Issue by anthonydvu** [soflyy/oxygen-bugs-and-features#765]
**Describe the feature you'd like to see included in Oxygen.**
Add ability to add styling options for guttenberg client view so client can change font size and padding as needed for existing sections. Integrate with original Guttenberg styling editor if possible or put right hand column styling options for client to be able to edit the following:
-padding
-margin
-font-styling (typography, size, weight)
-background section color 
-background image/overlay (image file + color)
- Allow add of additional section blocks BETWEEN oxygen designed sections via guttenberg editor

**What are the use cases for this feature?**

- Clients possibly wanting to change section styling, color, and images for specific sales and promotions (I.e. Fourth of july sale)
-Client chooses to put promo code or add additional section blocks for specific promotions.
-Client decides to make their own design changes within guttenberg
- Client decides to use guttenberg editor for adding additional blocks to integrate smoothly with Oxygen design (i.e. if padding and margin of newly added guttenberg block 

**Examples of this feature or functionality.**
- See Elementor section blocks client view styling functionality (except it would be nice for oxygen to integrate it with Guttenberg styling functionality.

**Comment by alriksson** [comment]
Do not agree, that could give clients too much free room to destroy things. But what I would like to see is a client mode manager:

Maybe one global and one on each gutenberg block.

Options to enable or disable styling options so you can have what you want enabled and I can disable. Would be disabled by default.

What I miss is that I can not really disable editing on text fields in the blocks the same way as dynamic data fields are disabled from editing. Some text fields I do not want client to be editing.

**Comment by anthonydvu** [comment]
I agree with this proposal to have each block have different role manager settings that allow clients to edit  what you allow them to edit. I would like to also have the ability to allow client to add guttenberg blocks  between the blocks created in oxygen. Obviously this would be an option that the agency has the option to enable/disable

**Comment by alriksson** [comment]
Do you mean creating design element rather than only full blocks? meaning that I can create 2 columns 60% - 40% and then the user can add headline and paragraphs from WP Core blocks in gutenberg into this design elements?

Thought I had a ticket about gutenberg design set blocks. But I haven't created that one, but here is some relevant ticket you may want to upvote:

https://github.com/soflyy/oxygen-bugs-and-features/issues/744
https://github.com/soflyy/oxygen-bugs-and-features/issues/671
https://github.com/soflyy/oxygen-bugs-and-features/issues/670
https://github.com/soflyy/oxygen-bugs-and-features/issues/669

**Comment by little-gatsby** [comment]
Most important issue in my opinion: the client should be able to change the background color of a section. This has basically no room for error and adds a decent amount of flexibility.
